modules/admin_manual/pages/installation/deployment_recommendations/nfs.adoc

```diff
@@ -0,0 +1,146 @@
+= Network File System (NFS) Deployment Recommendations
+:keywords: nfs, network file system, nfsv4, mtu, async, noasync
+:description: This guide covers the official ownCloud NFS (Network File System) deployment recommendations.
+:autofs-url: https://help.ubuntu.com/community/Autofs
+:nfs-read-write-delegations-url: https://tools.ietf.org/html/rfc7530#section-1.4.6
+:mount-man-page-url: http://man7.org/linux/man-pages/man8/mount.8.html
+:lockd-url: https://docs.oracle.com/cd/E19455-01/806-0916/rfsrefer-9/index.html
+:nfs-strong-security-architecture-url: https://tools.ietf.org/html/rfc7530#section-3 
+:rpc-statd-url: https://linux.die.net/man/8/rpc.statd
+:nfs-man-page-url: https://linux.die.net/man/5/nfs
+
+ownCloud recommends using NFS for any scenario other than local storage. 
+It has solid performance and is very stable.
+This document contains ownCloud’s official deployment recommendations.
+
+There can be different scenarios where ownCloud's storage is located on an NFS mount (primary/secondary). Also in some scenarios multiple application servers can use the same NFS mount.
+
+== Use NFSv4
+
+We recommend using NFSv4 over previous versions for a number of key reasons.
+These are:
+
+* *Improved Security:* It {nfs-strong-security-architecture-url}[mandates a strong security architecture]. It does not require {rpc-statd-url}[rpc.statd] or {lockd-url}[lockd]. As a result, it only uses port 2049.
+* *Improved Reliability:* Uses TCP by default instead of UDP.
+* *Improved Performance:* It uses Multi-Component Messages, which reduce network traffic. It is capable of using a 32KB page size, compared to the default, 1024 bytes.
+* Use of {nfs-read-write-delegations-url}[Read/Write Delegations].
+
+== Tune the Mount Parameters
+
+=== _netdev
+
+Use this option to ensure that the NFS storage is available before ownCloud starts.
+It is essential when MySQL’s data files are located on the NFS storage, as MySQL could error if the connection to the storage is not ready when ownCloud starts.
+
+TIP: Alternatively, you can use {autofs-url}[autofs] to ensure that mounts are always available.
+
+=== soft
+
+We recommend this option to better handle storage failures.
```

Comment:
Why do we recommend that?, An Administrator has to know how the system should be configured.

IMHO we should say: Soft mounts are easier to administrate and Hard, if you don't have enough experience this could be your option.

Reply:
I believe that that was in the notes in #256.

Reply:
I based it on the following, [which you wrote](https://github.com/owncloud/docs/issues/256#issuecomment-411718997).

> Soft-mount is recommended in case that a storage failure happens. otherwise turning off the owncloud server is more problematic with a hard mount and you need a boot and shutdown sequence in the servers.